src/components/Nav/Nav.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { GiHamburgerMenu } from 'react-icons/gi';
+import { GoSearch } from 'react-icons/go';
+import { BsHandbag, BsInstagram, BsYoutube } from 'react-icons/bs';
+import { useState } from 'react';
```

Comment:
```suggestion
import React, { useState } from 'react';
```
1번째 줄에서 위와 같이 한번에 import 할 수 있어 보이네요!